File: Polypaint/PolypaintApp/app/src/main/java/com/example/polypaintapp/Account.kt
package com.example.polypaintapp

class Account {

    constructor(firstName: String, lastName: String, username: String, password: String, imageData: ByteArray?, avatarUrl: String, firstTimeUser: Boolean,
                joinedRooms: Array<String>, stats: Stats) {

        this.firstName = firstName
        this.lastName = lastName
        this.username = username
        this.password = password
        this.imageData = imageData
        this.avatarUrl = avatarUrl
        this.firstTimeUser = firstTimeUser
        this.joinedRooms = joinedRooms
        this.stats = stats
    }

    var firstName: String
    var lastName: String
    var username: String
    var password: String
    var imageData: ByteArray?
    var avatarUrl: String
    var firstTimeUser: Boolean
    var joinedRooms: Array<String>
    var stats: Stats
}

File: Polypaint/PolyPaint/PolyPaint/Modeles/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolyPaint.Modeles
{
    class Account
    {
        public string firstName { get; set; }
        public string lastName { get; set; }
        public string username { get; set; }
        public string password { get; set; }
        public byte[] imageData { get; set; }
        public string avatarUrl { get; set; }
        private bool firstTimeUser_ { get; set; }
        public Stats stats;
        private static Account instance;
        public Account(string username, string password, string firstName = "", string lastName = "", byte[] imageData = null,
            string avatarUrl = null, Stats stats = null)
        {
            this.firstName = firstName;
            this.lastName = lastName;
            this.username = username;
            this.password = password;
            this.imageData = imageData;
            this.avatarUrl = avatarUrl;
            this.stats = stats;
        }

        public static Account Instance
        {
            get
            {
                if (instance == null)
                    instance = new Account("", "");
                return instance;
            }

            set { instance = value; }
        }
    }
}
